PyInstaller/hooks/hook-pkg_resources.py

```diff
@@ -20,3 +20,13 @@
 hiddenimports.append('pkg_resources.py2_warn')
 
 excludedimports = ['__main__']
+
+# Some more hidden imports. See:
+# https://github.com/pyinstaller/pyinstaller-hooks-contrib/issues/15#issuecomment-663699288
+# `packaging` can either be its own package, or embeded in
+# `pkg_resources._vendor.packaging`, or both.
+# Assume the worst and include both if present.
+hiddenimports += [
+    'packaging.version', 'packaging.specifiers', 'packaging.requirements',
```

Comment:
Wouldn't this be better as `collect_submodules('packaging')`?

Reply:
Probably. I only left it like that because I saw that `pkg_resources` explicitly lists them.

Reply:
Changed it